sdk/src/main/java/com/oursky/authgear/latte/Latte.kt

```diff
@@ -407,6 +409,58 @@ class Latte(
         }
     }
 
+    suspend fun migrateSession(
+        accessToken: String,
+        migrationEndpoint: String,
```

Comment:
Same as iOS SDK. This function should only take accessToken. The endpoint is configured at Latte initialization time.

Reply:
Updated at [9df4194](https://github.com/authgear/authgear-sdk-android/pull/202/commits/9df4194c9529a56c9e65130164a160690085f62f)